script/compile_run.sh

```diff
@@ -16,10 +16,9 @@ rm -f ../src/kern/exec.o
 bpftool btf dump file /sys/kernel/btf/vmlinux format c > ../src/kern/vmlinux.h
 kcc ../src/kern/exec.c
 mv exec.o ../src/kern/exec.o
-echo "Open a new terminal and use the following command: sudo cat /sys/kernel/debug/tracing/trace"
 
 bpftool gen skeleton ../src/kern/exec.o > ../src/user/exec.skel.h
 ucc ../src/user/user.c
 mv user ../src/user/user
 
-sudo ../src/user/user
+../src/user/user
```

Comment:
Why did you drop the sudo ?

Reply:
i have root permissions on the vm and sudo is an unknown command :D

Reply:
you can install sudo with apt install sudo

I understand that you happen to be root on your VM but most users won't, it's usually a bad practice to run everything as root (compiler etc...), your instructions should use the least privileges possible. Only the "user" binary needs to run as root so you should keep sudo here.